packages/@lwc/wds-playground/src/browser/ui/pg/playground/playground.js

```diff
@@ -28,6 +29,8 @@ export default class Playground extends LightningElement {
   ssrIsRendered = false;
   csrIsRendered = false;
   cacheBust = false;
+  layoutShiftAttributions = [];
```

Comment:
It looks as though `layoutShiftAttributions` are not reset at any time. So the list will just grow on subsequent renders.

Rather than taking this approach, the logic to enable & then disable collection of layout shift attributions should be part of `renderComponent`.

Reply:
addressed this